azure_functions_worker/dispatcher.py

```diff
@@ -299,12 +342,11 @@ def update_opentelemetry_status(self):
 
             self._context_api = context_api
             self._trace_context_propagator = TraceContextTextMapPropagator()
-            self._otel_libs_available = True
 
             logger.info("Successfully loaded OpenTelemetry modules. "
```

Comment:
We can remove this log in `update_opentelemetry_status` in favor of the one added in `initialize_azure_monitor`. It's redundant now

Reply:
Removed.